doc/sailfish-webview.qdocconf

```diff
@@ -0,0 +1,56 @@
+project         = Sailfish OS WebView Component
+description     = Sailfish OS Webview Component Reference Documentation
```

Comment:
Sailfish WebView Component

Reply:
Similar to Sailfish Silica and Sailfish Pickers for instance.

Reply:
Thanks; I've updated all instances to reflect this.